content/_changelogs/12.2.0.md

```diff
@@ -0,0 +1,47 @@
+## 12.2.0
+
+_Released 12/20/2022_
+
+**Features:**
+
+- Added the ability to match on `resourceType` with
+  [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept), and to see
+  the resource type of an intercepted request as `req.resourceType`. Addresses
+  [#14525](https://github.com/cypress-io/cypress/issues/14525).
+- Users working in React Component Testing projects can now generate a basic
+  spec file by selecting from the components that exist in their project.
+  Addresses [#24008](https://github.com/cypress-io/cypress/issues/24008).
+
+**Bugfixes:**
+
+- Fixed a regression introduced in
+  [Cypress 10.8.0](https://docs.cypress.io/guides/references/changelog#10-8-0)
+  where the `CYPRESS_EVERY_NTH_FRAME` environment variable was not being set
+  appropriately causing all frames to be captured which slowed down tests. Fixes
+  [#23830](https://github.com/cypress-io/cypress/issues/23830).
+- Fixed an issue where the `query` object was not available on requests from
+  [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept) once they
+  were yielded. Fixes
+  [#25088](https://github.com/cypress-io/cypress/issues/25088).
+- Fixed a regression introduced in
+  [Cypress 12](https://docs.cypress.io/guides/references/changelog#12-0-0) where
+  [`cy.get()`](https://docs.cypress.io/api/commands/get) would ignore a `null`
+  value for the 'withinSubject' option. Fixes
+  [#25104](https://github.com/cypress-io/cypress/issues/25104).
+- Fixed Typescript typings for
```

Comment:
Lets list this last-- its not as impactful as the of the other bug fixes in this list.